src/app/plugins/wolframalpha.plugin.ts

```diff
@@ -0,0 +1,78 @@
+import { MessageAttachment, MessageEmbed } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
+import Environment from '../../environment';
+// @ts-ignore
```

Comment:
why?

Reply:
wolfram api doesnt support es6 imports

Reply:
*It does but it doesn't have any type declarations so typescript doesn't like that.